prawns/balances/src/handlers/balances.balanceset.event.ts

```diff
@@ -0,0 +1,56 @@
+import { EventHandlerContext } from '@subsquid/substrate-processor';
+import { decodeAddress, getRegistry, getOrCreate, encodeAddress } from '../utils';
+import {
+  SubstrateBalanceAccount, SubstrateBalanceSet,
+  SubstrateNetwork,
+  SubstrateTreasuryAwarded,
+} from '../model';
+import { getBalancesBalanceSetEvent } from './typeGetters/getBalancesEvents';
+
+export default (network: SubstrateNetwork, tokenIndex: number) => {
+  return async (ctx: EventHandlerContext) => {
+
+    const depositor = ctx.extrinsic?.signer;
+    if (!depositor) return;
```

Comment:
If I'm not wrong, the signer is not the `depositor`, it's just the caller - which is the root.

Do we really need this check here? We don't even use the extrinsic or the signer.

I suggest to remove it or at most do:
```
if (ctx.extrinsic?.signer) return;
```

If you **really** wanna keep it.

Reply:
Good spot about the signer, it's kind of redundant here.

FYI, this check in our other handlers is more for TypeScript convenience, as the extrinsic isn't a required property on the event type, event though in all our use cases it is guaranteed to be there.

Reply:
Yep, I understand now this is redundant (by virtue of it being done under a sudo call) so will remove it. Cheers guys.

Reply:
@straticJeff, you ended up not removing it. Is it because you forgot to push? :D